### DISCUSSION TITLE
Renewing GitHub student pack with a different email

### DISCUSSION THREAD

**Discussion by bakayu** [community/community#56765] [GitHub Education] [answered]
### Hi everyone,

I enrolled into a diploma course and will be applying for GitHub student dev pack once I get my student id, but I'm planning on joining a Uni next year, is it possible to renew my student pack with a different student id (the one which I will get from the uni) or apply again whole together on the same account?
I am asking this for in case apply for the pack and do some work and make repos on this account then be unable to enjoy the student pack in Uni, would have to make another account then and the work on this account would kinda be wasted.
Thanks.

**Comment by mrepol742** [comment, marked as answer]
You can still apply for github student pack using different student id as long as it's yours, it's valid and proved you are still a student.

**Reply by bakayu** [reply]
thanks for confirming!
